### ISSUE TITLE
Unable to resume slideshow after keyboard nav

### ISSUE THREAD

**Issue by tristanjahier** [woothemes/FlexSlider#297]
Hello,

I found something quite anoying with the keyboard nav.
When navigating with the keyboard, it pauses the slideshow. And I am unable to resume it.

When PauseOnHover is set to true, you don't have this problem while using the direction nav or the control nav, since you trigger kinds of mouseover then mouseout events.

I found out a trick for that, but I think it is not the proper solution :

```
after: function(slider) {
	slider.trigger('mouseover'); slider.trigger('mouseleave');
}
```
